### PR TITLE
Adding telemetry tracking in PyTorch 1.5

### PR DESCRIPTION
**Pull request by abhinavs95**
Created a new PR because I lost access to the older one: https://github.com/aws/deep-learning-containers/pull/194

Changes: 
* Modified training docker files to include telemetry tracking script. Inference DLCs use public binaries so I have skipped those docker files.
* Updated PyTorch binaries (with telemetry tracking enabled) in s3 bucket. The URLs remain unchanged.
* Added a test to verify functionality.

### CODE REVIEW

File: test/dlc_tests/container_tests/bin/pytorch_tests/test_pt_dlc_telemetry_test [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+set -e
+
+echo "Verify pytorch telemetry test. You can follow progress on the log file : $TRAINING_LOG" | tee -a $TRAINING_LOG
+
+python ${BIN_DIR}/pytorch_tests/test_pt_dlc_telemetry_test.py > ${TRAINING_LOG} 2>&1
+
+if grep "DLC Telemetry perf test Passed" $TRAINING_LOG; then
+  echo "Successfully verified Telemetry perf test."
+else
+  echo "Telemetry perf test failed."
```
Comment (TusharKanekiDey) [inline test/dlc_tests/container_tests/bin/pytorch_tests/test_pt_dlc_telemetry_test:16]: Can u replace perf with performance in both the files.

File: test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py [hunk 2]
```diff
@@ -89,3 +90,13 @@ def test_pytorch_mpi(pytorch_training, ec2_connection, gpu_only, py3_only):
 @pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
 def test_nvapex(pytorch_training, ec2_connection, gpu_only):
     execute_ec2_training_test(ec2_connection, pytorch_training, PT_APEX_CMD)
+
+
+@pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
```
Comment (TusharKanekiDey) [inline test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py:95]: Can u make it run on only one GPU instance like this(https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py#L83)

File: test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py [hunk 3]
```diff
@@ -89,3 +90,13 @@ def test_pytorch_mpi(pytorch_training, ec2_connection, gpu_only, py3_only):
...
+def test_pytorch_telemetry_gpu(pytorch_training, ec2_connection, gpu_only):
+    execute_ec2_training_test(ec2_connection, pytorch_training, PT_TELEMETRY_CMD)
+
+
+@pytest.mark.parametrize("ec2_instance_type", PT_EC2_CPU_INSTANCE_TYPE, indirect=True)
```
Comment (TusharKanekiDey) [inline test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py:100]: Can u make it run on only one CPU instance like this(https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py#L89)

File: test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py [hunk 4]
```diff
@@ -89,3 +90,13 @@ def test_pytorch_mpi(pytorch_training, ec2_connection, gpu_only, py3_only):
 @pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
 def test_nvapex(pytorch_training, ec2_connection, gpu_only):
     execute_ec2_training_test(ec2_connection, pytorch_training, PT_APEX_CMD)
+
+
+@pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
+def test_tensorflow_telemetry_gpu(pytorch_training, ec2_connection, gpu_only):
```
Comment (ChaiBapchya) [inline test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py]: shouldn't it be pytorch?

**Comment by TusharKanekiDey** [comment]
Merged PR #248 which is created from this PR

**Pull request closed without merging**